src/features/MentorPage/MentorChips/MentorChips.tsx

```diff
@@ -42,6 +51,9 @@ const SkillChips = styled.div`
   &.show-more {
     height: fit-content;
   }
+  &.show-less {
+    height: 10.5rem;
+  }
```

Comment:
This is cool! But I would still prefer to pass the _shouldShowAllSkills_ as a prop to the SkillChips and then change the css-properties based on that value.

Reply:
Here I would like to see the prop passed to the styled-component like in the Chip

Reply:
When the shouldShowAllSkills is true, then height is fit-content, otherwise 10.5rem :p